library/StandardLibrary/src/Juvix/Library/LineNum.hs

```diff
@@ -15,4 +16,7 @@ instance A.ToJSON T where
 instance A.FromJSON T where
   parseJSON = A.genericParseJSON (A.defaultOptions {A.sumEncoding = A.ObjectWithSingleField})
 
-makeLensesWith camelCaseFields ''T
+instance Hashable T where
+  hash T {tLine, tCol} = hash (hash tLine, hash tCol)
+
+-- makeLensesWith camelCaseFields ''T
```

Comment:
Why remove the Lens here?

Reply:
This was breaking somehow and then I forgot about it.